src/web-ui/src/public/ProductDetail.vue

```diff
@@ -214,6 +215,22 @@ export default {
         let root_url = process.env.VUE_APP_IMAGE_ROOT_URL
         return root_url + this.product.category + '/' + this.product.image
       }
+    },
+    productStock: function () {
+      if (this.product.currentstock > 0 ) {
+        return "Items currently in stock : " + this.product.currentstock 
+      }
+      else {
+        return "Sorry this item is currently out of stock"
+      }
+    },
+    addToCartisDisabled: function () {
+      if (this.product.currentstock > 0 ) {
```

Comment:
Consider reducing function to single line `return !this.product.current_stock > 0` for brevity.